data/conf/llap/tez-site.xml

```diff
@@ -38,6 +38,10 @@
     <name>tez.runtime.shuffle.fetch.buffer.percent</name>
     <value>0.4</value>
   </property>
+  <property>
+    <name>tez.am.launch.cmd-opts</name>
+    <value> --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.util.concurrent=ALL-UNNAMED --add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.util.regex=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED </value>
```

Comment:
Looks like the same add-opens is added in all the config files. It can be made a constant system property variable in the pom.xml and referenced in config files